android/app/src/main/java/poke/rogue/helper/data/mapper/TypeMatchedResultMapper.kt

```diff
@@ -0,0 +1,12 @@
+package poke.rogue.helper.data.mapper
+
+import poke.rogue.helper.data.model.MatchedResult
+
+fun MatchedResult.toString(): String {
+    return when (this) {
+        MatchedResult.STRONG -> "강한"
+        MatchedResult.WEAK -> "약한"
+        MatchedResult.INVALID -> "무효한"
+        else -> ""
```

Comment:
1. INVALID 대신 `Void` 는 어떨까요? INVALID 는 `유효하지 않은` 이란 느낌이 강해서요!

2. enum class 를 사용할 때 `else` 문은 지양해야합니다. 
```suggestion
    return when (this) {
        MatchedResult.STRONG -> "강한"
        MatchedResult.WEAK -> "약한"
        MatchedResult.VOID -> "무효한"
        NORMAL -> "일반"
```

3. 개인적으로 `""` 가 넘어왔을 경우는 논리 에러라 판단됩니다. 빈 문자열로 넘어왔을 경우에 개발자가 에러가 발생했다는 것을 인지하기 힘들지 않을까요??

Reply:
`INEFFECTIVE` 도 좋네용

Reply:
넵 `INEFFECTIVE `로 변경할께용